picard/coverart/providers/__init__.py

```diff
@@ -17,30 +17,47 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+from picard import log
 from picard.plugin import ExtensionPoint
 
 
 _cover_art_providers = ExtensionPoint()
 
+# default order for providers
+_providers_order = [
+    'Cover Art Archive',
+    'Amazon',
+    'Whitelist',
+    'CaaReleaseGroup',
+    'Local',
+]
+
 
 def register_cover_art_provider(provider):
     _cover_art_providers.register(provider.__module__, provider)
 
 
-def cover_art_providers():
+def cover_art_providers(order=_providers_order):
```

Comment:
Isn't it enough to simply call `register_cover_art_provider` in the correct order?